en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
- 
- 
-![GPS](../../images/flight_modes/GPS_s.png) **RETURN: Return to Launch (RTL) / Return to Home**
 
-The aircraft returns to the home position and hovers in a circle above the home position at a preset altitude.
- 
-![GPS](../../images/flight_modes/GPS_s.png) **MISSION**
+#### Pause (Auto Pause)
 
-In Mission mode the aircraft will start the execution of a predefined Mission or Flight Plan. The Mission can be edited and uploaded to the aircraft with the software called Ground Control Station. The PX4 GCS is called QGroundControl and it can run on Windows, Linux, Android and iOS. You can download it from qgroundcontrol.com. The Mission upload to the aircraft can be done via USB connection on the ground or by wireless Data Link sometimes called Telemetry. During flight you can edit and change the current mission via this Data Link and monitor the current flight status and telemetry data live.
- 
-**TAKE OFF**
+You can use Pause in case of emergency or during a mission in order to Hold the aircraft at its current position autonomously, while you get time to access the situation either emergency or mission related. When you activate Pause mode with a pre-programmed switch the airplane will start circling around at fixed GPS position, holding its current altitude during the circles. During hold you are not able to control the craft with the RC. It is the first line of defense if you loose control over the aircraft.
 
-There are a couple of options for automatic Take Off for airplanes. We could perform take off from runway, from launch pad or throw the plane from hand. Different aspects from the take off process can be preset in the Prameters section in the PX4 autopilot.
 
-**Hand throwing**
+#### Return Mode
 
-When **Take Off** is activated the autopilot will give thrust to the motor/s and the props will start spinning. Then we throw the airplane up and forward, it gives throttle and starts climbing to e preset take off altitude. Once the altitude is reached the airplane starts circling at the current altitude and wait for pilot's command input.
+![GPS](../../images/flight_modes/GPS_s.png)  Vehicle returns to the launch/home position.
 
-**Launch Pad take off**
+The aircraft returns to the home position and hovers in a circle above the home position at a preset altitude.
 
-**Runway take off**
+#### Mission Mode
 
+![GPS](../../images/flight_modes/GPS_s.png)
 
-**LAND**
+In Mission mode the aircraft will start the execution of a predefined *mission* or flight plan. The mission can be edited and uploaded to the aircraft with a Ground Control Station (GCS).
 
+> **Tip** The main PX4 for working with PX4 is called [QGroundControl](https://docs.qgroundcontrol.com/en/).
 
-## VERTICAL TAKE OFF AND LANDING AIRCRAFT (VTOL)
+#### Takeoff Mode
```

Comment:
This is a useful summary but very incomplete. We do need this info, but I think detailed in https://docs.px4.io/en/flight_modes/takeoff.html and only a little here.